docs/handbook/getting-started.md

```diff
@@ -580,6 +575,15 @@ Replace "X" with the appropriate letter for your USB device
 
 ### illumos/Solaris
 
+
+#### Method 1 (New releases)
+
+```bash
+sudo dd bs=4M if=./OI_MATE_experimental.usb of=/dev/sdb status=progress && sync
```

Comment:
Perhaps, should mention GNU dd (/usr/gnu/bin/dd) vs illumos dd (/usr/bin/dd) differences